src/components/CytoscapeGraph/graphs/GridGraph.ts

```diff
@@ -0,0 +1,10 @@
+export class GridGraph {
+  static getLayout() {
+    return {
+      name: 'grid',
+      fit: false,
+      nodeDimensionsIncludeLabels: true,
+      padding: 0
```

Comment:
maybe add `animate: false` to show explicit intent.  Not a blocker.

Reply:
+1 I'll add it.